dev_guide/builds.adoc

```diff
@@ -629,7 +629,9 @@ located at `_<system-registry>_/_<namespace>_/ruby-20-centos7`. The
 `_<system-registry>_` is defined as a service with the name `docker-registry`
 running in OpenShift.
 
-. Define a build with a strategy that consumes the image stream:
+. Define a build with a trigger that points to the image stream. If the
```

Comment:
I still feel like it makes more sense to have the "If an image stream is the base image for the build" step before defining the triggers. The "If" makes it clearly circumstantial, so even if they're only going to define a trigger that monitors an arbitrary image stream it shouldn't trip them up and can just skip it. As is, it reads through kinda backwards.

@csrwng Do you feel strongly about the re-order?